### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by seeseemelk**
# Description
This adds a `CODEOWNERS` file limiting who can modify specific files.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Modifications to the source code should be handled by the code review team
+*.java                             @MockBukkit/code-reviewers
```
Comment (seeseemelk) [inline .github/CODEOWNERS]: This team got renamed
```suggestion
*.java                             @MockBukkit/project-members
```

**Comment by TheBusyBiscuit** [comment]
This seems familiar :D

https://github.com/Slimefun/Slimefun4/blob/master/.github/CODEOWNERS

**Comment by seeseemelk** [comment]
> This seems familiar :D

Pff... I have no idea what you're talking about 🤷

**Comment by TheBusyBiscuit** [comment]
You will probably also need to update the branch settings if you want code owner reviews to be required.
image

**Comment by seeseemelk** [comment]
> You will probably also need to update the branch settings if you want code owner reviews to be required.

Done

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Modifications to the source code should be handled by the code review team
+*.java                             @MockBukkit/project-members
+
+# Modifications to sensitive files should be reviewed by maintainers
+/.github/                          @MockBukkit/maintainers
+/build.gradle                      @MockBukkit/maintainers
+/settings.gradle                   @MockBukkit/maintainers
```
Comment (TheBusyBiscuit) [inline .github/CODEOWNERS:9]: Maybe also add `/gradle/wrapper/` here?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+/CONTRIBUTING.md                   @MockBukkit/maintainers
+/README.md                         @MockBukkit/maintainers
+/logo.png                          @MockBukkit/maintainers
```
Comment (TheBusyBiscuit) [inline .github/CODEOWNERS]: @HuffleAnimations has no access to the logo? Rude.

File: .github/CODEOWNERS (hunk 2 above)
Comment (TheBusyBiscuit) [reply .github/CODEOWNERS:9]: Also `/jitpack.yml` perhaps?

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (seeseemelk) [reply .github/CODEOWNERS:9]: Added them

File: .github/CODEOWNERS (hunk 3 above)
Comment (seeseemelk) [reply .github/CODEOWNERS]: She does not have write access to the project, and thus can't become a code owner.

**Review by TheBusyBiscuit — APPROVED** [review]
LGTM

File: .github/CODEOWNERS (hunk 3 above)
Comment (TheBusyBiscuit) [reply .github/CODEOWNERS]: sad

**Pull request merged**